src/arrayMethodReduce.js

```diff
@@ -5,7 +5,19 @@
  */
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let start = 0;
+    let feedBack = initialValue;
+
+    if (arguments.length === 1) {
```

Comment:
```suggestion
    if (initialValue === undefined) {
```
your version works well but is better when your condition strict